app/rpc/namespaces/eth/api.go

```diff
@@ -1032,11 +1032,14 @@ func (api *PublicEthereumAPI) generateFromArgs(args rpctypes.SendTxArgs) (*evmty
 	}
 
 	// Sets input to either Input or Data, if both are set and not equal error above returns
+	var data *hexutil.Bytes
 	var input []byte
```

Comment:
It is suggested to use `var input hexutil.Bytes` and remove `var data *hexutil.Bytes`